### PR TITLE
Alarm cleanup + tooltip and techreborn manual UI

### PR DESCRIPTION
**Pull request by jusvit**
Cleanup alarm code
add a tooltip telling users that they can change the sound
removed WIP tag from manual
added a UI for the manual with a button that takes them to the wiki and a button to join our discord(via link)
Added myself to contributors(?)

capture

### CODE REVIEW

File: src/main/resources/assets/techreborn/lang/en_us.lang [hunk 1]
```diff
@@ -718,6 +718,13 @@ techreborn.tooltip.inventory=Inventory
 techreborn.tooltip.upgrades=Upgrades
 techreborn.tooltip.transferRate=Transfer Rate
 techreborn.tooltip.tier=Tier
+techreborn.tooltip.alarm=Shift rightclick to change sound
+
+#ManualUI
+techreborn.manual.wiki=Online wikipedia
```
Comment (modmuss50) [inline src/main/resources/assets/techreborn/lang/en_us.lang]: Its not wikipedia, its just a wiki

File: src/main/java/techreborn/client/gui/GuiManual.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+		this.fontRenderer.drawString(text2, ((width / 2) - this.fontRenderer.getStringWidth(text2) / 2), height / 2 + 5, 4210752);
+		super.drawScreen(mouseX, mouseY, partialTicks);
+	}
+
+	@Override
+	protected void actionPerformed(GuiButton button) {
+		switch (button.id) {
+			case 1:
+				try {
+					Desktop.getDesktop().browse(new java.net.URI("http://wiki.techreborn.ovh"));
```
Comment (modmuss50) [inline src/main/java/techreborn/client/gui/GuiManual.java]: This needs to use GuiConfirmOpenLink, so it asks the user if they want to go to the url.

```java
this.mc.displayGuiScreen(new GuiConfirmOpenLink(this, "url", buttonID, false));
```

File: src/main/java/techreborn/client/gui/GuiManual.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+				} catch (Exception e) {
+					System.err.print(e);
+				}
+				break;
+			case 2:
+				try {
+					Desktop.getDesktop().browse(new java.net.URI("http://discord.gg/0tCDWb77cvetwm0e"));
```
Comment (modmuss50) [inline src/main/java/techreborn/client/gui/GuiManual.java]: Same here as well.

**Review by modmuss50 — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/techreborn/lang/en_us.lang (hunk 1 above)
Comment (jusvit) [reply src/main/resources/assets/techreborn/lang/en_us.lang]: isn't wiki short for Wikipedia?

Comment (modmuss50) [reply src/main/resources/assets/techreborn/lang/en_us.lang]: No, Wikipedia is the name of a wiki.

**Review by modmuss50 — APPROVED** [review]
OK, this looks better.

**Pull request merged**